st2common/st2common/models/db/stormbase.py

```diff
@@ -78,6 +78,23 @@ class StormBaseDB(StormFoundationDB):
     }
 
 
+class StormContentDB(StormBaseDB):
+    """Abstraction for a user content model from a pack."""
+    name = me.StringField(required=True)
+    pack = pack = me.StringField(
+        required=False,
+        help_text='Name of the content pack.',
+        unique_with='name')
+    file_uri = me.StringField(
```

Comment:
Looking at this action models need to use artifact_uri for the python script or the yaml in workflows. 

meta_uri seems better than file_uri